src/handlers/issue/generate-permits.ts

```diff
@@ -242,7 +242,9 @@ function generateDetailsTable(totals: TotalsById) {
         const commentScore = commentScores[index];
 
         const commentUrl = commentSource.comment.html_url;
-        const truncatedBody = commentSource ? commentSource.comment.body.substring(0, 64).concat("...") : "";
+        const truncatedBody = commentSource
+          ? commentSource.comment.body.replaceAll("<", "&lt;").replaceAll(">", "&gt;").substring(0, 64).concat("...")
```

Comment:
Does that html entity render correctly and do what is expected? Have you tested it inside of GitHub comments? Not sure if the UI has any special rules around these html entities.

Reply:
Bit of a blind fix as it is the same inside the new conversation rewards version which is tested and works. It should not affect any render as it just escapes characters for proper HTML reading. I will try to setup a test scenario.